CMakeLists.txt

```diff
@@ -2176,6 +2176,9 @@ endif()
 #
 
 # Battery meter
+#
+# The battery meter is only available on Linux, macOS and Windows, therefore
+# this option is forcibly set off on all other platforms.
 cmake_dependent_option(BATTERY "Battery meter support" ON "WIN32 OR UNIX" OFF)
```

Comment:
If we use default option, it will be a normal option that is ON by default on Windows/MacOS/Linux and OFF by default on other OSes.

With cmake_dependent_option, it's ON by default on Windows/macOS/Linux and simply does not exist on other OSes (it won't show up when using `cmake -L ..` and it will always be off).

Reply:
Ping: Why not "WIN32 OR UNIX OR APPLE"?

Reply:
`APPLE` ∈`UNIX`.

To match Linux/BSD, you need to do `UNIX AND NOT APPLE`.

Reply:
https://cmake.org/cmake/help/latest/variable/UNIX.html#unix

Reply:
:laughing: